ui/src/flux/apis/index.ts

```diff
@@ -44,53 +42,75 @@ export const getAST = async (request: ASTRequest) => {
   }
 }
 
-export interface GetTimeSeriesResult {
+export interface GetRawTimeSeriesResult {
   didTruncate: boolean
-  tables: FluxTable[]
+  csv: string
   uuid?: string
 }
 
-export const getTimeSeries = async (
+// Returns the CSV file directly from the Flux service
```

Comment:
my personal preference is to keep a comment around, since I like shorter variable names and feel that a comment is easier to understand than some long camelCased function name. as for stale comments, I think that's the responsibility of whomever changes this function in the future.